src/index.html

```diff
@@ -18,5 +22,85 @@
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="/"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Company Logo"
+        />
+      </a>
+      <nav>
+        <ul class="nav_list">
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. Consider renaming `nav_list` to `nav__list` to follow BEM naming convention and to indicate that it's a part of the `nav` block.